src/bin/cargo/commands/remove.rs

```diff
@@ -0,0 +1,122 @@
+use cargo::core::dependency::DepKind;
+use cargo::ops::cargo_remove::remove;
+use cargo::ops::cargo_remove::RemoveOptions;
+use cargo::ops::resolve_ws;
+use cargo::util::command_prelude::*;
+use cargo::util::toml_mut::manifest::DepTable;
+
+pub fn cli() -> clap::Command<'static> {
```

Comment:
> Deferred work

Forgot to call out that besides any FCP, we'll need to make sure we merge this far enough away from branching a new beta for these to be implemented.